### DISCUSSION TITLE
Explore on Android App stopped working

### DISCUSSION THREAD

**Discussion by CodeBleu** [community/community#127562] [Mobile]
This has been an issue for a few weeks now, so I assume the issue happened in the last few releases or maybe API changes.

"Something went wrong"

"something went.wrong.while executing your query ......"

I cant even uploaded a screenshot as it gives an error...ugh

Would be nice if when it pops up the error, I could actually select the big ID it produces and copy that text.  It says to pass that along, but can't upload screenshot, and can't easily copy that ID to paste here.

**Comment by Fabri-dev** [comment]
I have the same problem

**Comment by Arrowar** [comment]
![Screenshot_20250110_223236_GitHub.jpg](https://github.com/user-attachments/assets/2ca77d37-a218-4800-abb2-a1c4ae3fd83a)

Same error.
